src/screens/Learn/Landing/Landing.tsx

```diff
@@ -9,33 +9,48 @@ import {
 } from '../Learn.style';
 import AppMetaTags from 'components/AppMetaTags/AppMetaTags';
 import SectionButton, { ButtonTheme } from './SectionButton';
-import TableOfContents, { Item } from 'components/TableOfContents';
+// import TableOfContents, { Item } from 'components/TableOfContents';
 import { Anchor } from 'components/TableOfContents';
+import { formatNumericalDate } from 'common/utils';
 import { LandingSection, landingPageContent } from 'cms-content/learn';
 
-const header = landingPageContent.header;
-const intro = landingPageContent.intro;
-const sections = landingPageContent.sections;
+/*
+  Commenting out all things related to the table of contents, since we only have 2 items as of now.
+*/
 
 const Landing = () => {
-  function getSectionItems(sections: LandingSection[]): Item[] {
-    return sections.map(section => ({
-      id: section.sectionId,
-      title: section.sectionTitle,
-    }));
-  }
+  const {
+    header,
+    intro,
+    sections,
+    metadataTitle,
+    metadataDescription,
+  } = landingPageContent;
+
+  // function getSectionItems(sections: LandingSection[]): Item[] {
```

Comment:
table of contents feels a little strange when its only for 2 items, so i commented it out for now. will un-comment if peeps disagree and want to keep it // will remove the code if peeps agree and want to nix it for now